client/gutenberg/editor/edit-post/components/sidebar/block-sidebar/style.scss

```diff
@@ -11,11 +11,11 @@
 		color: $dark-gray-500;
 	}
 
-	&:first-child {
-		margin-top: 16px;
-	}
-
-	&:last-child {
+	&.editor-block-inspector__advanced {
 		margin-bottom: -16px;
 	}
 }
+
+.editor-block-inspector__card {
```

Comment:
🤔 I wonder if we should move these overrides up to the [top style.css](https://github.com/Automattic/wp-calypso/blob/master/client/gutenberg/editor/style.scss#L17). When we update packages we'll likely copy paste from the edit post folder, and we'll likely regress.

Reply:
Makes sense, I'll move this before shipping.